README.md

```diff
@@ -8,12 +8,12 @@
 
 fastrace is a tracing library [10~100x faster](#benchmarks) than others:
 
-![benchmark](etc/img/head-benchmark.svg)
+![benchmark](https://raw.githubusercontent.com/fast/fastrace/refs/heads/main/etc/img/head-benchmark.svg)
 
 Features:
 
 - [Extremely fast](#benchmarks)
-- [Rich features for logging](fastrace/examples/logging.rs)
+- [Rich features for logging](https://github.com/fast/fastrace/blob/main/fastrace/examples/logging.rs)
```

Comment:
iIRC this path should work. Have you checked it?

Reply:
seems not working on crates.io

Reply:
> iIRC this path should work. Have you checked it?

No, it is not working. 🥲